docs/guide/translations.md

```diff
@@ -4,7 +4,7 @@ title: Translations
 parent: Guide
 ---
 
-# Translations (experimental)
+# Translations
 
 To use experimental support for `I18n` translations, include `ViewComponent::Translatable`:
```

Comment:
I think we'll need to rework this section a bit 😄

Reply:
For sure! 😉